ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/components/DeleteModal.tsx

```diff
@@ -41,15 +39,7 @@ function DeleteModal({
                 </Button>,
             ]}
         >
-            {error && (
-                <Alert
-                    isInline
-                    variant={AlertVariant.danger}
-                    title={error}
-                    className="pf-u-mb-sm"
-                />
-            )}
-            <p>{children}</p>
```

Comment:
We're removing the Alert from the `DeleteModal` because we need to craft the error messages with regard to bulk deletions